dev/devicelab/lib/tasks/plugin_tests.dart

```diff
@@ -370,6 +370,7 @@ public class $pluginClass: NSObject, FlutterPlugin {
           destination: 'platform=macOS',
           configuration: 'Debug',
           testName: 'native_plugin_unit_tests_macos',
+          extraOptions: <String>['-parallel-testing-enabled', 'NO'],
```

Comment:
This parameter is not present in this branch. Please update this pr to have passing presubmits.